docs/01-app/02-guides/upgrading/codemods.mdx

```diff
@@ -24,6 +24,78 @@ Replacing `<transform>` and `<path>` with appropriate values.
 
 ## Codemods
 
+### 16.0
+
+#### Migrate from `next lint` to ESLint CLI
+
+##### `next-lint-to-eslint-cli`
+
+```bash filename="Terminal"
+npx @next/codemod@latest next-lint-to-eslint-cli .
```

Comment:
```suggestion
npx @next/codemod@canary next-lint-to-eslint-cli .
```
Let's encourage `canary` here similar to our upgrade guides to get any fixes we push without having to wait for a stable.